app/assets/v2/js/abi.js

```diff
@@ -1,23 +1,23 @@
 
 
 var bounty_address = function (){
     switch(document.web3network){
         case "mainnet":
-            return '0x4e1799e6e54d848394b0b90593e789e939a9f801';
+            return '0x066128b9f7557b5398db3d4ed141f2e64245ffa1';
         break;
         case "ropsten":
-            return '0x3102118ba636942c82d1a6efa2e7d069dc2d14bd';
+            throw "not supported";
         break;
         case "kovan":
             throw "not supported";
         break;
         case "rinkeby":
-            throw "not supported";
+            throw "0xd5ea74ad5bcec859ea7a7104401ea1c746c84074";
         break;
         case "custom network":
-            return '0x0ed0c2a859e9e576cdff840c51d29b6f8a405bdd';
+            return '0xbe716ab501a10d9ea0362bf20a9b77aba1bd2d88';
```

Comment:
What is the custom network in terms of Standard Bounties?  From what I understand only mainnet and rinkeby is supported right now.

Reply:
custom network is `testrpc`

testrpc just a way to create a localhost ethereum network => https://www.npmjs.com/package/ethereumjs-testrpc

highly HIGHLY recommend this approach for testing and quick iteration.. ephemeral `testrpc` nodes are cheap and very easy to develop upon

Reply:
Is that address the always the same for a custom network though?  It would vary based on the test environment right?

Reply:
depends on how you start testrpc.

i use this script , https://github.com/gitcoinco/smart_contracts/blob/master/scripts/testrpc.bash , which uses the same seed every single launch.  thus saving me the extra step of having to update this everytime i deploy a change